src/Helpers/Configuration.ts

```diff
@@ -0,0 +1,11 @@
+import { keys, fromPairs, map, pipe } from 'ramda'
+
+import { camelCaseToScreamingSnakeCase } from './camelCaseToScreamingSnakeCase'
+
+export const toPair = (s: string) => [camelCaseToScreamingSnakeCase(s), s]
```

Comment:
don't need to export this to test, its just an implementation detail of `createEnvToConfigurationKeyMap`